tests/operator/operator.go

```diff
@@ -888,11 +893,11 @@ spec:
 			Eventually(fetchVirtHandlerCommand, 60*time.Second, 5*time.Second).Should(ContainSubstring(maxDevicesCommandArgument))
```

Comment:
This is not related to this PR, but don't you think that this could be tested equally well in a unit test?

Reply:
maybe, but what's the downside of checking the virt-handler command contains `--max-devices 10`  in an e2e?